src/ui/views/ViewContactUs.qml

```diff
@@ -17,12 +17,12 @@ Item {
     id: contactUsRoot
 
```

Comment:
Let's replace lines 315 and 316  of this file with to vertically recenter the Thank you! view 
```                
anchors.verticalCenter: parent.verticalCenter
anchors.verticalCenterOffset: (Theme.rowHeight + Theme.vSpacing) * -1